web/react/pages/channel.jsx

```diff
@@ -70,8 +69,8 @@ function setupChannelPage(props) {
     // Modals
     //
     ReactDOM.render(
-        <GetLinkModal />,
-        document.getElementById('get_link_modal')
+        <GetTeamInviteLinkModal />,
```

Comment:
React-Bootstrap does something similar, but there was a problem with having modals that closed themselves and opened other modals in there place. There's probably better ways to do this and we might go back and revisit it at some point when we move the last few modals away from using plain bootstrap